app/controllers/v0/sessions_controller.rb

```diff
@@ -37,19 +37,21 @@ def persist_session_and_user!
       @session = Session.new(user_attributes.slice(:uuid))
       @current_user = User.find(@session.uuid) || create_new_user
       @session.save && @current_user.save
+      async_create_evss_account(@current_user)
     end
 
     def user_attributes
       attributes = @saml_response.attributes.all.to_h
       {
-        first_name:   attributes['fname']&.first,
-        middle_name:  attributes['mname']&.first,
-        last_name:    attributes['lname']&.first,
-        zip:          attributes['zip']&.first,
-        email:        attributes['email']&.first,
-        ssn:          attributes['social']&.first&.delete('-'),
-        birth_date:   parse_date(attributes['birth_date']&.first),
-        uuid:         attributes['uuid']&.first,
+        first_name:     attributes['fname']&.first,
+        middle_name:    attributes['mname']&.first,
+        last_name:      attributes['lname']&.first,
+        zip:            attributes['zip']&.first,
+        email:          attributes['email']&.first,
+        ssn:            attributes['social']&.first&.delete('-'),
+        birth_date:     parse_date(attributes['birth_date']&.first),
+        uuid:           attributes['uuid']&.first,
+        last_signed_in: Time.zone.now,
```

Comment:
Shouldn't that be `Time.current.utc`? (I find best practices around timezones confusing.)